src/legacy/ui/public/notify/notifier.js

```diff
@@ -87,19 +87,11 @@ function restartNotifTimer(notif, cb) {
 
 const typeToButtonClassMap = {
   danger: 'kuiButton--danger', // NOTE: `error` type is internally named as `danger`
-  info: 'kuiButton--primary',
-};
-const buttonHierarchyClass = index => {
```

Comment:
This function was removed because `kuiButton--primary` conflicts with `kuiButton--secondary`